ari4.py

```diff
@@ -14,7 +14,7 @@
 
 
 m = re.compile(r'[kK][eE][kK]')
-bgbscanner = re.compile(r'whos .*b')
```

Comment:
needs to only do 3 letters, and end after the 3 letters. someone asked who babe boss was (a very good question) and this didnt work.